maps/sccv_horizon/code/sccv_horizon_areas.dm

```diff
@@ -173,6 +173,12 @@
 	area_blurb = "The scents of oil and machine lubricant fill the air in this workshop."
 	area_blurb_category = "robotics"
 
+/area/operations/lower/machinist/surgicalbay
+	name = "Machinist Surgical Bay"
+	icon_state = "machinist_workshop"
+	area_blurb = "The scent of sterilized equipment mixed with anxiety and concern fill the air."
```

Comment:
Do not railroad feelings into player characters like this.
Why would every machinist feel "anxiety and concern" every single round they enter this room, even if just for a moment?
Why would any IPC or robot feel "anxiety and concern" even if they're just coming to get their lightly damaged arm or leg fixed?

Reply:
Edited